README.md

```diff
@@ -1,23 +1,30 @@
-# ![Jazz Logo](misc/logo.png) Jazz Serverless Platform
+# ![Jazz Logo](misc/logo.png) Jazz Serverless Development Platform
 
 [![Build Status](https://travis-ci.org/tmobile/jazz.svg?branch=master)](https://travis-ci.org/tmobile/jazz)
+<a href="https://github.com/tmobile/jazz/graphs/contributors" alt="Contributors"><img src="https://img.shields.io/github/contributors/tmobile/jazz.svg" /></a>
 [![License](https://img.shields.io/badge/License-Apache%202.0-yellowgreen.svg)](https://github.com/tmobile/jazz/blob/master/LICENSE)
+![GitHub release](https://img.shields.io/github/release/tmobile/jazz.svg)
 [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/TMO-OSS/Jazz)
 [![Slack Chat](https://img.shields.io/badge/Chat-Slack-ff69b4.svg)](https://tmo-oss-getinvite.herokuapp.com/)
 
 **Seamlessly build, deploy & manage cloud-native serverless applications!**
 
 ## Introduction
 
-Jazz, a serverless platform, accelerates adoption of serverless technology within your enterprise. Jazz comes with a beautiful UI that lets developers quickly create serverless applications with a click of a button. Its modular design makes it easy to add new integrations.
+Jazz, a serverless development platform, accelerates adoption of serverless technology within your enterprise. Jazz comes with a beautiful UI that lets developers quickly create serverless applications with a click of a button. Its modular design makes it easy to add new integrations that your enterprise needs.
```

Comment:
Should we call it beautiful UI? Developers want Functional UI.

Reply:
I want `Beautiful` & `Functional` UI :)

Reply:
:)